javascript/jquery/security/audit/jquery-insecure-selector.yaml

```diff
@@ -49,7 +49,7 @@ rules:
             }
       - pattern-either:
         - pattern: |
-            $(<... $Y ...>);
+            $(<... $Y ...>)
         - pattern: |
             $("..." + (<... $Y ...>));
```

Comment:
Should we remove this semi-colon too then?

Reply:
yep, I will remove them too